examples/v1alpha1/pipelineruns/conditional-pipelinerun-continue-after-skip-ordering.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Condition
+metadata:
+  name: always-false
+spec:
+  check:
+    image: alpine
+    script: |
+      exit 1 
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: echo-expected
+spec:
+  steps:
+    - name: echo-expected
+      image: ubuntu
+      script: 'echo expected'
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: echo-unexpected
+spec:
+  steps:
+    - name: echo-file-exists
+      image: ubuntu
+      script: 'echo UNEXPECTED!'
```

Comment:
maybe we could make this task exit with a non-zero exit code, so that if it wasn't skipped the pipeline would fail? not important but just wondering :D